qiskit/transpiler/passes/scheduling/alap.py

```diff
@@ -34,15 +33,16 @@ class ALAPSchedule(BaseSchedulerTransform):
         removed in a future release.
```

Comment:
This note looks redundant now <img width="624" alt="Screenshot 2023-05-10 at 22 52 03" src="https://github.com/Qiskit/qiskit-terra/assets/766693/b1d7f5fd-3ad6-4c91-8d74-0010dcb9e95a">